docs/legacy/high-availability.asciidoc

```diff
@@ -1,6 +1,9 @@
 [[high-availability]]
 === High Availability
 
+IMPORTANT: These commands only apply to the APM Server binary installation method.
+Fleet-managed users should see {fleet-guide}/fleet-agent-proxy-support.html[Fleet/Agent proxy support].
+
```

Comment:
This section actually holds true for both - the apm server binary and Fleet managed. 

The link to the Fleet/Agent proxy is still valuable, but it actually describes how to configure a proxy between Elastic components (from EA to ES, to Fleet Server, to pacakge registry). Whereas this page focuses on adding a proxy in front of the APM Server for high availability. 

My suggestion to change to something like:
> When using the Fleet-managed apm server, you might also be interested in taking a look at {fleet-guide}/fleet-agent-proxy-support.html[Fleet/Agent proxy support].